Framework/HistogramData/inc/MantidHistogramData/Histogram.h

```diff
@@ -0,0 +1,462 @@
+#ifndef MANTID_HISTOGRAMDATA_HISTOGRAM_H_
+#define MANTID_HISTOGRAMDATA_HISTOGRAM_H_
+
+#include "MantidHistogramData/DllConfig.h"
+#include "MantidKernel/cow_ptr.h"
+#include "MantidHistogramData/BinEdges.h"
+#include "MantidHistogramData/BinEdgeStandardDeviations.h"
+#include "MantidHistogramData/BinEdgeVariances.h"
+#include "MantidHistogramData/Points.h"
+#include "MantidHistogramData/PointStandardDeviations.h"
+#include "MantidHistogramData/PointVariances.h"
+#include "MantidHistogramData/Counts.h"
+#include "MantidHistogramData/CountVariances.h"
+#include "MantidHistogramData/CountStandardDeviations.h"
+#include "MantidHistogramData/Frequencies.h"
+#include "MantidHistogramData/FrequencyVariances.h"
+#include "MantidHistogramData/FrequencyStandardDeviations.h"
+
```

Comment:
Minor point, but based on "include what you" use best practices, I think you're incorrectly obtaining type definitions through loosely related includes. For example: No explicit include of `HistogramX.h`, but that comes through via `BinEdges.h`.